README.md

```diff
@@ -3,10 +3,9 @@ base-images-docker
 Source for the Google-maintained base docker images. These images can be used as a base for container-based application development.
 
 Images currently supported:
-* Debian 8 (Jessie)
-* Debian 9 (Stretch)
-* Ubuntu
-* Node.js
+* Debian 8 (Jessie) `launcher.gcr.io/google/debian8` and `gcr.io/google-appengine/debian8`
+* Debian 9 (Stretch) `launcher.gcr.io/google/debian9` and `gcr.io/google-appengine/debian9`
+* Ubuntu `gcr.io/gcp-runtimes/ubuntu_16_0_4:latest`
```

Comment:
launcher.gcr.io/google/ubuntu16_04 is also valid

Reply:
done!